documentation/CLI_usage.md

```diff
@@ -0,0 +1,195 @@
+# CLI Usage
+
+---
+## npd
+
+### Description
+napari plugin devtools scans the current python environment to find out details about napari plugin information.
+
+### Usage
+```
+npd [COMMANDS] [ARGS...]
+```
+
+### Commands
+|Command        |Description                                          |
```

Comment:
Love the table and new format - easy to read and understand what commands are available.
Also like that you split out the discover command from the validate.

Reply:
+1